mitodl/bootcamp-ecommerce
# Payment history page

#### What are the relevant tickets?
Fixes #624 

#### What's this PR do?
Adds payment history page for an application

#### How should this be manually tested?
Click the 'View statement' link on the application page. It should resemble the mockup and not have any errors.

#### Screenshots
The [mockup](https://projects.invisionapp.com/d/main?origin=v7#/console/19928472/418348650/preview?scrollOffset=595.5) for reference. The screenshot does not strictly match it at the moment:

![Screenshot from 2020-06-12 18-05-19](https://user-images.githubusercontent.com/863262/84549983-57d8e600-acd7-11ea-89c5-f9a340fe8833.png)


![Screenshot from 2020-06-11 19-01-41](https://user-images.githubusercontent.com/863262/84447747-e5a0cc80-ac16-11ea-8f6d-8399bbd35fee.png)
